labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+
+<html>
+
+<head>
+
+<link rel="stylesheet" href="stylesheet.css">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:800" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+</head>
+
+<body>
+
+<div class="main">
+  <header>
+  <p>chirpus inc.</p>
+  </header>
```

Comment:
The wordmark AND the nav make up the header; seeing just the workmark in a paragraph being called the header is unexpected.